elementary/clients/slack/slack_message_builder.py

```diff
@@ -122,6 +122,20 @@ def create_button_action_block(text: str, url: str) -> dict:
             ],
         }
 
+    @staticmethod
+    def create_section_with_button(
```

Comment:
you don't need this one if you're not reformatting slack (this is intended to make the link be a button and not just hyperlink text)

Reply:
Yeah but I don't mind adding this function as well because it is a generic one and could be used with the message builder everywhere 🙂